backend/battling/tests/test_forms.py

```diff
@@ -105,3 +110,69 @@ def test_create_team_form_with_invalid_pokemon(self):
         )
 
         self.assertFalse(form.is_valid())
+
+
+class PokemonApiIntegrationTest(TestCaseUtils):
```

Comment:
This class should not exist since you are testing the form here. You should put this inside `CreateTeamFormTest`